geode-core/src/main/java/org/apache/geode/management/internal/cli/i18n/CliStrings.java

```diff
@@ -1662,6 +1662,9 @@
   public static final String LIST_INDEX__STATS = "with-stats";
   public static final String LIST_INDEX__STATS__HELP =
       "Whether statistics should also be displayed.";
+  public static final String LIST_INDEX__MAINT = "maint-status";
```

Comment:
Same as before, spelling out the whole word would be my preference

Reply:
Thanks. I will update.